deploy/kubernetes/helm/templates/bookie.yaml

```
@@ -154,7 +154,7 @@ spec:
             - name: ledgers-disk
               mountPath: /bookkeeper/data/ledgers
 
-{{- if or (eq .Values.platform "aws") (eq .Values.platform "baremetal") }}
+{{- if .Values.mountLocalDisk }}
```

This name is ok, but to keep consistent with the other name change I suggested, let's make this one `.Values.bookkeeper.useHostPath`.
```suggestion
{{- if .Values.bookkeeper.useHostPath }}
```